L/Libcerf/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Libcerf"
+version = v"1.14"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v$version/libcerf-v$version.tar.gz", "065346b3360943c9961517f8c49ae13fe956835f6fc3b53e9d307e41feec3a34")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libcerf-v$version/
```

Comment:
Since this a raw string, I don't think `$version` will be defined here. I suggest this:
```suggestion
cd libcerf-*/
```